build/Uno.UI.Build.csproj

```diff
@@ -47,7 +47,8 @@
 		<Message Text="Building for $(Configuration) and $(Platform) BuildReason:$(BUILD_REASON) Version:$(GitVersion_SemVer) UNO_UWP_BUILD:$(UNO_UWP_BUILD)" />
 
 		<CallTarget Targets="BuildT4Generator" />
-		<CallTarget Targets="UpdateFileVersions;UpdateTasksSHA;PrepareNuGetPackage" Condition="$(_isWindows)" />
+		<CallTarget Targets="UpdateFileVersions;UpdateTasksSHA" Condition="$(_isWindows)" />
```

Comment:
@jeromelaban Why do we need this condition?

Reply:
Yes, packages do not need to be built for non-windows targets, which is what you've been seeing when enabling errors.

Reply:
Is there something specific that makes building packages on non-Windows hard?
Anyway, I guess the change done here to separate `PrepareNuGetPackage` unconditionally shouldn't be problematic right?

Reply:
It should not be a problem no, once the case sensitivity issues are adjusted.